jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/MessagePublisher.java

```diff
@@ -0,0 +1,50 @@
+package us.dot.its.jpo.ode.coder;
+
+import java.text.ParseException;
+import java.time.ZonedDateTime;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import us.dot.its.jpo.ode.OdeProperties;
+import us.dot.its.jpo.ode.model.OdeBsmData;
+import us.dot.its.jpo.ode.model.OdeData;
+import us.dot.its.jpo.ode.model.OdeObject;
+import us.dot.its.jpo.ode.util.DateTimeUtils;
+import us.dot.its.jpo.ode.wrapper.J2735BsmSerializer;
+import us.dot.its.jpo.ode.wrapper.MessageProducer;
+
+public class MessagePublisher {
+
+   private static final Logger logger = LoggerFactory.getLogger(MessagePublisher.class);
+   private OdeProperties odeProperties;
+   protected MessageProducer<String, OdeObject> objectProducer;
+
+   public MessagePublisher(OdeProperties odeProps) {
+      this.odeProperties = odeProps;
+      this.objectProducer = new MessageProducer<>(odeProperties.getKafkaBrokers(),
+            odeProperties.getKafkaProducerType(), null, J2735BsmSerializer.class.getName());
```

Comment:
We will be publishing OdeBsmData objects. SO this should be using a OdeBsmDataSerializer, right?